spec/features/user/user_requests_their_password_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+describe "user" do
+  it "user receives sms with password reset code" do
+    user = User.create!(
+           first_name: "Ames",
+           last_name: "James",
+           email: "ames@sample.com",
+           cellphone: "720-770-2071",
```

Comment:
We decided to store our phone numbers as only numbers (without the dashes). This may break a future test to format the phone number.